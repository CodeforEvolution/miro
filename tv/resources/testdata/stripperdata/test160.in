<a href="http://blip.tv/file/get/Commoncourtesy-Toronto185.mov"><img src="http://blip.tv/file/get/Commoncourtesy-Toronto185.mov.jpg" /></a><div class="blip_description"><p>We stayed with Frizelle in her wonderful apartment.&#160; Thanks Sarah, we had a fabulous time!</p><p>&#160;We\'re in Minnesota now, staying with James Payne at the Franconia Sculpture park.</p>
