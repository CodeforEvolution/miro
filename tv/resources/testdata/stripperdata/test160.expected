(u"We stayed with Frizelle in her wonderful apartment.\xa0 Thanks Sarah, we had a fabulous time!\n\xa0We\\'re in Minnesota now, staying with James Payne at the Franconia Sculpture park.", [(0, 0, u'http://blip.tv/file/get/Commoncourtesy-Toronto185.mov')])
